launchmon/src/sdbg_rm_map.hxx

```diff
@@ -290,6 +290,8 @@ public:
 
   bool is_attfifo_sup();
 
+  bool is_cont_on_att();
```

Comment:
Can this commit be squashed with 3b215a6b536d79f2395057bcd04aab92604d1662 as well?